### PR TITLE
Changed so now there is reduced blinking in the Side Panel annimation.

### PR DESCRIPTION
**Pull request by dc-andysign**
---
name: Side panel infinite blinking fix
about: Side panel arrow reduced blinking
---

### Prerequisites

[x] FAQs for common solutions checked
[x] Checked that your issue isn't already filed

### Description

The animation added to the side panel grey arrow goes on endlessly moving away the focus from the overall design. This reduces the animation to only 10 loops

### Steps to Reproduce

Go to SidePanel 

### Versions

N/A

### Additional Information

Before:

Screen Recording 2024-02-25 at 21 55 22

After:

image

---

### CODE REVIEW

File: src/Navigation/SidePanel/common.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const Button = styled('div')`
   transition: all 0.5s ease-out-in;
   top: 10%;
   left: ${({ style: { width } }: any) => width || 300}px;
-  animation: indicate 1s infinite;
+  animation: indicate 1s 10;
```
Comment (mattdean-digicatapult) [inline src/Navigation/SidePanel/common.tsx:20]: I'd suggest just keeping it simple and removing the animation. t's graphically unpleasent

Comment (dc-andysign) [reply src/Navigation/SidePanel/common.tsx:20]: Are you sure ? the obvious implication is a big diff churn ;)

Comment (mattdean-digicatapult) [reply src/Navigation/SidePanel/common.tsx:20]: yes I'm sure

Comment (dc-andysign) [reply src/Navigation/SidePanel/common.tsx]: Done. ( i think this is now ready to be merged )

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**